Dockerfile.autobuild

```diff
@@ -5,13 +5,34 @@
 # all of it's assets, there are more optimal ways to do this but this is the
 # most friendly and has everything contained in a single instance.
 #
-# Authors: Isaac Bythewood
-# Updated: Mar 29th, 2016
+# Authors: Isaac Bythewood, Jason Kaltsikis
+# Updated: May 2nd, 2020
 # Require: Docker (http://www.docker.io/)
 # -----------------------------------------------------------------------------
 
-# Base system is the LTS version of Ubuntu.
-FROM python:3.7-stretch
+# Build static yarn file
+FROM node:10.20.1-alpine3.11 as yarn-build
+
+WORKDIR pinry-spa
+COPY pinry-spa/package.json .
+RUN yarn install
+COPY pinry-spa .
+RUN yarn build
+
+
+# Required for other database options
+FROM python:3.7-slim-stretch as base
+
+RUN apt-get update \
+    && apt-get -y install gcc default-libmysqlclient-dev
+RUN pip --no-cache-dir install --user mysqlclient cx-Oracle
```

Comment:
We install all requirements to system's python package location, what's the reason to install package to user's home in docker env?

Personally I'd like to keep them in system-default package location.
I have no strong concern on installing as user's package or system wide package but I'd like keep all of the packages at the same location.

If you want to install these packages to user's directory, please also install all of the packages at the same location.

Reply:
We install the required packages in user's home so we can pass them in the final image. This way the image is less smaller than if we pass all the packages etc.